topdown/tokens.go

```diff
@@ -1064,18 +1064,28 @@ func builtinJWTDecodeVerify(bctx BuiltinContext, args []*ast.Term, iter func(*as
 	}
 	// RFC7159 4.1.4 exp
 	if exp := payload.Get(jwtExpKey); exp != nil {
-		// constraints.time is in nanoseconds but exp Value is in seconds
-		compareTime := ast.FloatNumberTerm(float64(constraints.time) / 1000000000)
-		if ast.Compare(compareTime, exp.Value.(ast.Number)) != -1 {
-			return iter(unverified)
+		switch exp.Value.(type) {
```

Comment:
NOTE: This issue fixes #5165, but I forgot to end the commit message with that!

Let's talk to determine the least painful way to get that resolved depending on whether anyone has pulled my commit.

Reply:
Making updates in your own fork is OK — you can just amend the commit and push -f 🙂

Reply:
OK, I'll do this when squashing after PR is approved.

Reply:
LGTM. You can squash right away as that will discard the approval anyway.